README.md

```diff
@@ -12,9 +12,10 @@
 # It's recommanded to use virtual environment
 python -m pip install --upgrade virtualenv
 virtualenv .venv
+source .venv/bin/activate
 
 python -m pip install -e . uvicorn
-source .venv/bin/activate
+export TEST_ENVIRONMENT=true  # set this when running locally to mock redis
```

Comment:
This one is a question rather than a comment, I see we use `fakeredis` - I think that's fine for unit tests etc. But if you're standing up the API locally, would it be better we recommended real redis? 🤷 I don't know!

Reply:
Not sure whether this is a deliberate decision or just a hack for the first MVP.

Sounds like material for a separate ticket.

Reply:
Makes sense!